crates/biome_service/src/explain/rules.rs

```diff
@@ -0,0 +1,1683 @@
+//! Generated file, do not edit by hand, see `xtask/codegen`
```

Comment:
The generated code was really badly formatted when I directly used docs string in `RuleMetadata`. Best way I figured out was to generate constants for them and use that instead, but there is surely a better way to fix that